scripts/cbmc_json_parser.py

```diff
@@ -329,6 +329,7 @@ def postprocess_results(properties, extra_ptr_check):
     has_failed_unwinding_asserts = has_check_failure(properties, GlobalMessages.UNWINDING_ASSERT_DESC)
     has_reachable_undefined_functions = modify_undefined_function_checks(properties)
     properties, reach_checks = filter_reach_checks(properties)
+    properties = filter_sanity_checks(properties)[0]
```

Comment:
Is there any chance this is empty?

Reply:
also, you should move this logic to be inside filter_sanity_checks

Reply:
sure, thanks!